doxygen/doxygen.cfg

```
@@ -2353,7 +2350,7 @@ INTERACTIVE_SVG        = YES
 # found. If left blank, it is assumed the dot tool can be found in the path.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-DOT_PATH               = /usr/bin/dot
+DOT_PATH               =
```

Here it may be good to comment out the dot path so it knows it's not an option we want to set 

```suggestion
#DOT_PATH               =
```